packages/common/src/hooks/useDialog/useDialog.tsx

```diff
@@ -20,6 +20,7 @@ export interface ModalProps {
   nextButton?: React.ReactElement<{
     onClick: () => Promise<boolean>;
   }>;
+  slideAnimation?: boolean;
```

Comment:
this one was neat 😉 
The slide animation make the 'OK & Next' functionality look great - but.. it does cause an issue with the Popper component used by the autocomplete, if using the `open` prop 🙄 
With the animation, the popper shows in the correct place vertically, but on the hard left of the screen! any interaction with the page fixes it.. but it is super weird. 

So, have added the option to disable the animation.

Reply:
oops. not my intention to remove it completely. 1ea2dbdb